src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -1172,7 +1172,7 @@ ss::future<cluster::init_tm_tx_reply> tx_gateway_frontend::limit_init_tm_tx(
             if (ec != tx_errc::none) {
                 vlog(
                   txlog.trace,
-                  "do_expire_old_tx with tx_id={} returned ec={}",
+                  "do_expire_old_tx with tx.id={} returned ec={}",
```

Comment:
nit: can we rename variables in this method to have more context so reader knows how `tx_id` differs from `tx.id` ?